sklego/neighbors.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.neighbors import KernelDensity
+from sklearn.utils import check_X_y
+from sklearn.utils.multiclass import unique_labels
+from sklearn.utils.validation import check_is_fitted, check_array, FLOAT_DTYPES
+
+
+# noinspection PyPep8Naming,PyAttributeOutsideInit
+class BayesianKernelDensityClassifier(BaseEstimator, ClassifierMixin):
+    """
+    Bayesian Classifier that uses Kernel Density Estimations to generate the joint distribution
+    Parameters:
+        - bandwidth: float
+        - kernel: for scikit learn KernelDensity
+    """
+    def __init__(self, bandwidth=0.2, kernel='gaussian'):
+        # TODO (2/6/2020) add all other sklearn kwargs
```

Comment:
yes ... definitely would like to see that :)

Reply:
Done son